Assets/Scripts/Player.cs

```diff
@@ -3,26 +3,33 @@
 
 public class Player : Entity
 {
-    private void FixedUpdate()
-    {
-        MovePlayer();
-    }
+  private void FixedUpdate()
+  {
+    MovePlayer();
+  }
 
-    private void MovePlayer()
-    {
-        // Set keyboard
-        Keyboard kb = Keyboard.current;
+  private void MovePlayer()
+  {
+    // Set keyboard
+    Keyboard kb = Keyboard.current;
 
-        // Return if no keyboard detected
-        if (kb == null)
-            return;
+    // Return if no keyboard detected
+    if (kb == null)
+      return;
 
-        // If user presses UP arrow or W key, player moves up
-        if (kb.upArrowKey.IsPressed() || kb.wKey.IsPressed())
-            transform.position = new Vector2(transform.position.x, transform.position.y + speed * Time.deltaTime);
+    // If user presses UP arrow or W key, player moves up
+    if (kb.upArrowKey.IsPressed() || kb.wKey.IsPressed())
+      transform.position = new Vector2(transform.position.x, transform.position.y + speed * Time.deltaTime);
+
+    // If user presses DOWN arrow or S key, player moves down
+    if (kb.downArrowKey.IsPressed() || kb.sKey.IsPressed())
+      transform.position = new Vector2(transform.position.x, transform.position.y - speed * Time.deltaTime);
+
+    if (kb.numpad0Key.IsPressed())
```

Comment:
Can you replace `numpad0Key` with `oKey` instead? I just realized the letter O in the game design doc looks a lot like a zero, so that's my bad! My laptop doesn't even have a numpad 😂